packages/platform/platform-multer/src/common/decorators/multipartFile.ts

```
@@ -0,0 +1,91 @@
+ * @param name
+ * @param maxCount
+ * @returns Function
+ * @decorator
+ * @input
+ */
+export function MultipartFile(name: string, maxCount?: number): ParameterDecorator {
+  return (...args: DecoratorParameters): void => {
+    const [target, propertyKey, index] = args;
+    const multiple = Metadata.getParamTypes(target, propertyKey)[index as number] === Array;
+
+    name = (typeof name === "object" ? undefined : name)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)